packages/client/tests/server/handlers.ts

```diff
@@ -608,4 +608,44 @@ export const handlers = [
             );
         },
     ),
+
+    rest.get<never, never>(
+        `${SERVER_ENDPOINT}/authentication/me`,
+        (_req, res, ctx) => {
+            const authenticationToken = localStorage.getItem('token');
```

Comment:
❔ questio: Should we implem the msw mocks as ready to receive both web auth and api token auth ?
Indeed atm client integ tests looks like they are only running on a IOS like OS. But this could change ?

Reply:
Great question, let's discuss about it.